README.md

```diff
@@ -1 +1,247 @@
+
+- **Compare Atlases:** BrainKB also integrates knowledge from diverse knowledge platform services if available for integration, providing the feature to compare knowledge from across different atlases (e.g., Allen Brain Atlases). 
+
+- **Find/correct Errors:** BrainKB will provide a feature to search existing knowledge and correct errors if any. 
+
+- **Add information/API:** BrainKB offers an API endpoint that enables seamless integration with its platform. These endpoints facilitate data ingestion from various sources, such as CSV files or raw text, for constructing KGs, performing search operations, and conducting analyses on the stored KGs.
+
+- **Doing meta-analysis:** Meta-analysis is a knowledge-intensive task that requires significant time and effort to find related studies, identify evidence items, annotate the contents, and aggregate the results [1]. BrainKB, which stores knowledge from diverse data sources, including scientific publications, facilitates the meta-analysis.
+
+	[1] _Tiddi, I., Balliet, D. and ten Teije, A., 2020. Fostering scientific meta-analyses with knowledge graphs: a case-study. In The Semantic Web: 17th International Conference, ESWC 2020, Heraklion, Crete, Greece, May 31–June 4, 2020, Proceedings 17 (pp. 287-303). Springer International Publishing._
+
+
+
+
+## Models
+- [GARS](https://github.com/brain-bican/models/blob/main/linkml-schema/gars.yaml)
+- [ANSRS](https://github.com/brain-bican/models/blob/main/linkml-schema/ansrs_core.yaml)
+- [Biolink](https://github.com/brain-bican/models/blob/main/linkml-schema/bican_biolink.yaml)
+- CAP/CAS
+
+
+## Usage Scenario
+**Actor:** Alice (Neuroscientists)
+
+**Task:** Alice wants to know if they can gain new insights from their newly collected neuroscience data.
+
+**Precondition:** The dataset is usable, i.e., is not corrupted and is related to the neuroscience domain.
+
+**Flow:**
+
+1. Alice uploads the data into the BrainKB platform through the BrainKB UI (User Interface).
+2. BrainKB, the system, then analyzes data. If any error, e.g., unsupported file format, it will return the error; otherwise, the system will proceed to the next step of knowledge extraction.
+3. The system will perform the knowledge extraction, validation, and alignment operation. If the validation or the alignment issue cannot be resolved automatically, the extracted knowledge represented via KG is flagged for expert review. Upon the successful review, the KGs are integrated (or stored) in the BrainKB storage and is available for visualization and analysis.
+
+**Postcondition:** Alice discovers new insights through the integration of diverse knowledge sources represented in BrainKB's KGs.
+
+## Technology
+- FastAPI
+- Docker
+- Serverles
```

Comment:
```suggestion
- Serverless
```

I will propose codespell ... ;-)

Reply:
Thanks, will fix it. Probably, I should stop using notepad.